dist/linux/byedpi.service

```diff
@@ -3,7 +3,8 @@ Description=byedpi
 Documentation=https://github.com/hufrea/byedpi
 
 [Service]
-ExecStart=ciadpi --split 1 --disorder 3+s --mod-http=h,d --auto=torst --tlsrec 1+s
+EnvironmentFile=/etc/default/byedpi
+ExecStart=ciadpi $OPTIONS
```

Comment:
Probably using something like `$BYEDPI_OPTIONS` will be better to make sure there is no problems with global environment variables.